bindings/python/crocoddyl/multibody/frames.cpp

```diff
@@ -56,13 +56,16 @@ void exposeFrames() {
   bp::class_<FrameMotion>("FrameMotion",
                           "Frame motion describe using Pinocchio.\n\n"
                           "It defines a frame motion (tangent of SE(3) point) for a given frame ID",
-                          bp::init<FrameIndex, pinocchio::Motion>(bp::args("self", "frame", "oMf"),
-                                                                  "Initialize the frame motion.\n\n"
-                                                                  ":param frame: frame ID\n"
-                                                                  ":param oMf: Frame motion w.r.t. the origin"))
+                          bp::init<FrameIndex, pinocchio::Motion, bp::optional<pinocchio::ReferenceFrame> >(
+                              bp::args("self", "frame", "oMf", "reference"),
+                              "Initialize the frame motion.\n\n"
+                              ":param frame: frame ID\n"
+                              ":param oMf: Frame motion w.r.t. the origin\n"
+                              ":param reference: Reference frame (default LOCAL)"))
```

Comment:
similarly, when you see the name "reference", it means the vector that should be given. Use the name reference_frame please.